tests/lib/__init__.py

```diff
@@ -250,10 +251,10 @@ def __init__(self, base_path, *args, **kwargs):
         # workaround for https://github.com/pypa/virtualenv/issues/306
         if hasattr(sys, "pypy_version_info"):
             lib = os.path.join(venv, 'lib-python', pyversion)
-        self.venv_path = venv
-        self.lib_path = lib
-        self.include_path = include
-        self.bin_path = bin
+        self.venv_path = Path(venv)
```

Comment:
These were already made `Path` objects, a couple of lines above

    venv, lib, include, bin = map(Path, path_locations)